software/controller/platformio.ini

```diff
@@ -58,6 +58,7 @@ build_flags =
 build_unflags =
   -std=gnu++11
   -std=gnu++14
+  -std=c++11
```

Comment:
This is now needed in addition to gnu++11? But not c++14?
In fact, I thought we were even using 17 or even 20 features?

Reply:
These are *un*-flags, meaning we specifically remove those from flags that are automatically added by platformio. I only added the one that platformio adds, and we need to remove (it caused build failures).